frontend-ui/src/utils/offliner.ts

```diff
@@ -68,63 +71,48 @@ export function buildCommandWithout(config: ExpandedScheduleConfig) {
 interface SingleScheduleDuration {
   single: boolean
```

Comment:
Must be `single: true` ; this will allow to get rid of `formattedMinDuration`, `formattedMaxDuration`, `minWorker` and `maxWorker` on this interface, and `formattedDuration` on `MultipleScheduleDuration`. Please clean this, it makes code extra hard to read.